pkg/monitoring/alloy/templates/monitoring-config.yaml.template

```diff
@@ -53,6 +53,7 @@ alloy:
     - secretRef:
         name: {{ .SecretName }}
     resources:
+      limits: null
```

Comment:
Why are you settings ressources here ? We agreed a while back to now remove limits though 🤔

Reply:
I guess this was done before we agreed on this. I just found out this bug now.

Reply:
I'm.just wondering why you're even settings requests hère. We should either use vpa/hpa or app defaults right?

Reply:
That's because the default resources are too low for the AlloyMetrics use case, and there was not VPA back then.

Reply:
https://github.com/giantswarm/observability-operator/pull/123